homeassistant/components/roborock/__init__.py

```diff
@@ -167,11 +168,17 @@ async def setup_device(
 
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
-    """Handle removal of an entry."""
+    """Handle unloading of an entry."""
     if unload_ok := await hass.config_entries.async_unload_platforms(entry, PLATFORMS):
         release_tasks = set()
         for coordinator in hass.data[DOMAIN][entry.entry_id].values():
```

Comment:
For a follow-up PR, switch from `hass.data[DOMAIN][entry.entry_id]` to `entry.runtime_data` like #117261